qldbdriver/integration_base_test.go

```diff
@@ -83,11 +84,23 @@ func (testBase *testBase) deleteLedger(t *testing.T) {
 			testBase.logger.Log("Encountered resource not found", LogInfo)
 			return
 		}
-		testBase.logger.Log("Encountered error during deletion", LogInfo)
-		testBase.logger.Log(err.Error(), LogInfo)
-		t.Errorf("Failing test due to deletion failure")
-		assert.NoError(t, err)
-		return
+		var riu *types.ResourceInUseException
+		if errors.As(err, &riu) {
+			if strings.Contains(riu.ErrorMessage(), "Ledger is being created") {
```

Comment:
If the ledger is still being created then L79 would have silently failed
```
➜ aws qldb update-ledger --name error-test --no-deletion-protection

An error occurred (ResourcePreconditionNotMetException) when calling the UpdateLedger operation: Ledger is not yet active
```

We should be OK since L68 creates the ledger without deletion protection, so L79 isn't really necessary.

Reply:
Removed in cd7927f.